<launch>
  <!-- Arguments -->
  <arg name="cmd_vel_topic" default="/diffdrive_controller/cmd_vel" />
  <arg name="odom_topic" default="diffdrive_controller/good_odom" />
  <arg name="move_forward_only" default="false"/>

  <!-- move_base -->
  <node pkg="move_base" type="move_base" respawn="false" name="move_base" output="screen">
    <param name="base_local_planner" value="dwa_local_planner/DWAPlannerROS" />
    <rosparam file="$(find FullMech_Rep)/param/costmap_common_params.yaml" command="load" ns="global_costmap" />
    <rosparam file="$(find FullMech_Rep)/param/costmap_common_params.yaml" command="load" ns="local_costmap" />
    <rosparam file="$(find FullMech_Rep)/param/local_costmap_params.yaml" command="load" />
    <rosparam file="$(find FullMech_Rep)/param/global_costmap_params.yaml" command="load" />
    <!-- <rosparam file="$(find FullMech_Rep)/param/move_base_params.yaml" command="load" /> -->
    <!-- <rosparam file="$(find FullMech_Rep)/param/dwa_local_planner_params_$(arg model).yaml" command="load" /> -->
    <remap from="cmd_vel" to="$(arg cmd_vel_topic)"/>
    <remap from="odom" to="$(arg odom_topic)"/>
    <!-- <param name="DWAPlannerROS/min_vel_x" value="0.0" if="$(arg move_forward_only)" /> -->
  </node>
</launch>